volatility3/framework/plugins/mac/mount.py

```diff
@@ -12,7 +12,7 @@
 
 class Mount(plugins.PluginInterface):
     """A module containing a collection of plugins that produce data typically
-    foundin Mac's mount command"""
+    founding Mac's mount command"""
```

Comment:
Hehehe, you did indeed find a typo, but what we'd been meaning to write was `found in` rather than `founding`.  5:)

Reply:
Oh, thank you. I think my English was not good enough. 😅